.github/workflows/countline.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+
+import os
+import glob
+
+#get current working dir, set count, and select file delimiter
+path = os.getcwd()
+#os.chdir(path)
+
+
+#parses through files and saves to a dict
+names={}
+pathName = path + '/**/*.dart'
+for fn in glob.glob(pathname = pathName, recursive = True):
```

Comment:
Use [os.path.walk](https://www.tutorialspoint.com/python/os_walk.htm) not glob.

Reply:
Done